travis/spark.conf

```diff
@@ -15,7 +9,7 @@ hibench.yarn.executor.num     1
 hibench.yarn.executor.cores   2
 
 # executor and driver memory in standalone & YARN mode
-spark.executor.memory  4g
+spark.executor.memory  1g
```

Comment:
Why changing this?

Reply:
@gczsjdy  Setting spark.executor.memory to 4g,  travis will fail  due to memory problem.  I will rollback it as we talked about.